main.go

```diff
@@ -104,7 +107,14 @@ func main() {
 		os.Exit(1)
 	}
 
+	downloaderURL, err := url.Parse(config.DownloaderURL)
```

Comment:
Any reason in particular for us attempting to parse this config URL, where we don't for the others?